src/lava/magma/compiler/compiler.py

```diff
@@ -251,6 +254,35 @@ def _group_proc_by_model(proc_map: PROC_MAP) \
 
         return grouped_models
 
+    # TODO: (PP) This currently only works for PyPorts - needs general solution
+    # TODO: (PP) Currently does not support 1:many/many:1 connections
+    @staticmethod
+    def _map_var_port_class(port: VarPort,
+                            proc_groups: ty.Dict[ty.Type[AbstractProcessModel],
+                                                 ty.List[AbstractProcess]]):
+        """Derives the port class of a given VarPort from its source RefPort."""
```

Comment:
'Derives' is the wrong word. The function literally maps something to something else. Say that and explain why it's doing this.

Reply:
Done